### ISSUE TITLE
GeoJSON driver: impossible to set RFC7946 option

### ISSUE THREAD

**Issue by aborruso** [duckdb/duckdb_spatial#246]
Hi,
if apply this command to natural earth data

```
duckdb -c "copy (select POSTAL,geom from st_read('./data/ne_110m_admin_0_countries.shp')) TO 'tmp.geojson' WITH (FORMAT GDAL, DRIVER 'GeoJSON',LAYER_CREATION_OPTIONS 'RFC7946=YES')"
```

I have this error

```
Error: IO Error: GDAL Error (1): PROJ: proj_create: unrecognized format / unknown name
```

How to set RFC7946 option?

Thank you

**Comment by Maxxen** [comment]
Hi! I am unable to reproduce this on DuckDB v0.9.2. What version are you using? 

The GDAL docs says:
> Coordinates must be geographic over the WGS 84 ellipsoid, hence if the spatial reference system specified at layer creation time is not EPSG:4326, on-the-fly reprojection will be done by the driver.

The problem is that DuckDB does not automatically supply a spatial reference system as DuckDB currently does not store that information as part of the geometry type. You would expect GDAL to use the default unknown projection when no CRS is supplied, alas the GeoJSON driver throws an error instead. But I think this has been fixed since #141.

However, as I can't reproduce this Im not sure but I think you could maybe work around this for now by simply passing a SRS explicitly to st_write. e.g. 
```sql
WITH (FORMAT 'GDAL', DRIVER 'GeoJSON', LAYER_CREATION_OPTIONS 'RFC7946=YES', SRS 'EPSG:4326')
```

**Comment by aborruso** [comment]
> Hi! I am unable to reproduce this on DuckDB v0.9.2. What version are you using?

I have updated to 0.9.2 and it works.

Thank you very much
